ai-service/src/ai_service/main.py

```
@@ -1,111 +1,60 @@
-def answer_question(user_question: str, top_k: int = 3) -> None:
-    # Step 1: Embed the user question
-    question_embedding = embedder.embed_text(user_question)
-    # Step 2: Query ChromaDB for relevant code snippets
-    results = db.query_chunks(question_embedding, number_of_results=top_k)
 
-    # Step 3: Prepare context for the LLM
-    documents = results.get("documents", [[]])
+# FastAPI exception handlers
+@app.exception_handler(errors.AIServiceError)
+async def ai_service_error_handler(
+    request: Request, exc: errors.AIServiceError
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)